.pre-commit-config.yaml

```diff
@@ -44,7 +44,7 @@ repos:
         args: [ --fix, --exit-non-zero-on-fix ]
       - id: ruff
         args: [ --select, "I", --fix, --exit-non-zero-on-fix ]
-        files: '^python/ray/serve/|^python/ray/train|^python/ray/data|^python/ray/_private/|^python/ray/llm/|^python/ray/tune/|^python/ray/dag/'
+        files: '^python/ray/serve/|^python/ray/train|^python/ray/data|^python/ray/_private/|^python/ray/llm/|^python/ray/tune/|^python/ray/includes/|^python/ray/internal/|^python/ray/ray_operator/|^python/ray/scripts/|^python/ray/serve/generated/serve_pb2.py|^python/ray/streaming/|^python/ray/dag/'
```

Comment:
Do we need `python/ray/serve/generated/serve_pb2.py` here given we have `^python/ray/serve/`